frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/ConfigContent.test.tsx

```diff
@@ -2,35 +2,51 @@ import React from 'react';
 import { ConfigContent } from './ConfigContent';
 import { render, screen } from '@testing-library/react';
 import { textMock } from '../../../../../../testing/mocks/i18nMock';
+import userEvent from '@testing-library/user-event';
 import type { BpmnContextProps } from '../../../contexts/BpmnContext';
 import { BpmnContext } from '../../../contexts/BpmnContext';
 import type { BpmnDetails } from '../../../types/BpmnDetails';
 import { BpmnTypeEnum } from '../../../enum/BpmnTypeEnum';
-import userEvent from '@testing-library/user-event';
 import type Modeler from 'bpmn-js/lib/Modeler';
 import { type BpmnTaskType } from '../../../types/BpmnTaskType';
 import { BpmnConfigPanelFormContextProvider } from '../../../contexts/BpmnConfigPanelContext';
+import type { BpmnApiContextProps } from '../../../contexts/BpmnApiContext';
+import { BpmnApiContext } from '../../../contexts/BpmnApiContext';
+import type { LayoutSets } from 'app-shared/types/api/LayoutSetsResponse';
 
 const mockBPMNXML: string = `<?xml version="1.0" encoding="UTF-8"?></xml>`;
 const mockAppLibVersion8: string = '8.0.3';
 
-const mockId: string = 'testId';
+const mockTaskId: string = 'testId';
 const mockName: string = 'testName';
+const layoutSetIdToUpdate: string = 'layoutSet1';
+
+const modelerRefMock = {
+  current: {
+    get: () => {},
+  } as unknown as Modeler,
```

Comment:
Is it necessary to cast twice here? 😄

Reply:
Yes, in this case I think it is.. 🙈 Since we cant mock all the 32 required props in the Modeler type